Rakefile

```diff
@@ -11,11 +13,35 @@ file PKGDIR do
   mkdir PKGDIR
 end
 
-file "#{BUILDDIR}/modules" do |t|
+file "#{BUILDDIR}/modules" => BUILDDIR do |t|
   if Dir["modules/*"].empty?
     sh "librarian-puppet install --verbose --path #{BUILDDIR}/modules"
+    # Rake::Task["prune_foreman_modules"].execute
   else
-    cp_r "modules/", BUILDDIR
+    cp_r "modules", "#{BUILDDIR}/modules"
+  end
+end
+
+desc 'Remove foreman modules from $BUILDDIR/modules'
+task :prune_foreman_modules => [] do
+  begin
+    # raise "No foreman modules found in #{FOREMAN_MODULES_DIR}"
+    if Dir["#{FOREMAN_MODULES_DIR}/*"].empty?
+      temp_dir = Dir.mktmpdir
+      Dir.chdir(temp_dir) do
+        puts 'Downloading foreman modules...'
+        `git clone https://github.com/theforeman/foreman-installer.git`
```

Comment:
Good point, it didn't came to my mind we would want to build against older foreman. I'll add variable to handle this to the rake file